l1-contracts/deploy-scripts/upgrade/DefaultEcosystemUpgrade.s.sol

```diff
@@ -208,20 +212,39 @@ contract DefaultEcosystemUpgrade is Script, DeployL1Script {
         addresses.bridges.l1NullifierImplementation = deploySimpleContract("L1Nullifier", false);
         addresses.bridges.l1AssetRouterImplementation = deploySimpleContract("L1AssetRouter", false);
         addresses.vaults.l1NativeTokenVaultImplementation = deploySimpleContract("L1NativeTokenVault", false);
+        addresses.bridges.erc20BridgeImplementation = deploySimpleContract("L1ERC20Bridge", false);
```

Comment:
Why do we need to deploy this? Only imports changed. Or you want to redeploy every contract?

Reply:
I thought about it a bit, we have lots of other contracts that have barely been modified and still redeployed on the default upgrade. An example being [L1NativeTokenVault](https://github.com/matter-labs/era-contracts/pull/1419/files#diff-288893da09eb3917a797035bfcb80ea7250a94d5c38278e3a576f76d0dd76fd7). 

If we are going the direction where we try to minimize forgetting updates, then it is how we should do it IMO